bsdlib/doc/gnss_extension.rst

```diff
@@ -145,15 +150,63 @@ The elevation mask parameter controls the elevation threshold angle (in degrees)
 Satellites with elevation angle less than the threshold are automatically excluded from the PVT estimation.
 
 Use case
+--------
+
    | Socket option:  :c:type:`NRF_SO_GNSS_USE_CASE`
    | Datatype:       :c:type:`uint8_t`
-   | Default         1
-   | Allowed values: 0, 1
+   | Default:        :c:type:`NRF_GNSS_USE_CASE_MULTIPLE_HOT_START` | :c:type:`NRF_GNSS_USE_CASE_NORMAL_ACCURACY`
+   | Allowed values:
+
+   | :c:type:`NRF_GNSS_USE_CASE_SINGLE_COLD_START`
+   | :c:type:`NRF_GNSS_USE_CASE_MULTIPLE_HOT_START`
+   | :c:type:`NRF_GNSS_USE_CASE_NORMAL_ACCURACY`
+   | :c:type:`NRF_GNSS_USE_CASE_LOW_ACCURACY`
+
+This bitmask is used to specify the combination of the target GNSS performance (single cold or multiple hot start) and the accuracy (normal or low).
+
+Low accuracy mode
++++++++++++++++++
+
+If you enable the low accuracy mode, the GNSS receiver demonstrates a looser acceptance criterion for a fix.
+The error in position calculation, when compared to the actual position, can be larger than the error in normal accuracy mode.
+In addition, in the low accuracy mode, the GNSS receiver might use only three satellites to determine a fix.
+In normal accuracy mode, four or more satellites are used.
+
+A position fix will be valid for a time duration of 24 hours. For a possible position fix using three satellites, one of the following criteria must be satisfied initially:
```

Comment:
The first sentence here is a bit confusing to me, I'm probably interpreting it wrongly. Is the meaning that a position fix will be valid for 24 hours in the context of the first bullet point below?

Reply:
I Jan Tore is right, this could be missleading. I suggest changing the text to:
"For a possible position fix using three satellites, one of the following criteria must be satisfied initially. When the criteria is met it will be valid for a time duration of 24 hours:"

Reply:
```suggestion
For a possible position fix using three satellites, one of the following criteria must be satisfied initially and when the criterion is met, it will be valid for a time duration of 24 hours:
```